packages/snowpack-loader/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import path from "path"
+import { getConfig } from "@lingui/conf"
+import { createCompiledCatalog, getCatalogs, getCatalogForFile } from "@lingui/cli/api"
+
+type LinguiConfigOpts = {
+  cwd?: string;
+  configPath?: string;
+  skipValidation?: boolean;
+}
+type SnowpackLoadOpts = {
+  filePath: string
+}
+function extractLinguiMessages(_?, linguiConfig: LinguiConfigOpts = {}) {
+  const strict = process.env.NODE_ENV !== 'production'
+  const config = getConfig(linguiConfig)
+
+  return {
+    name: '@lingui/snowpack-loader',
+    resolve: {
+      input: ['.po'],
```

Comment:
Probably here we should accept also .json files as webpack loader does.

Reply:
Sure